cpp/src/parquet/metadata.cc

```diff
@@ -312,6 +312,20 @@ class ColumnChunkMetaData::ColumnChunkMetaDataImpl {
     }
   }
 
+  inline std::optional<IndexLocation> GetColumIndexLocation() const {
```

Comment:
`inline` is implied if the method is defined inside the `class` body, so it's superfluous here.

Reply:
Fixed